### DISCUSSION TITLE
need help with support

### DISCUSSION THREAD

**Discussion by davidpalmspringd** [OrcaSlicer/OrcaSlicer#7313] [Q&A]
Using either tree or normal support, when the moidel prints the support can't be separated from the model. Is there a setting that makes it easier?

**Comment by sworcester001** [comment]
You have probably figured this out, but play with the top Z distance under >Process>Support>Advanced. There is a fine line between sticking too much and leaving a horrible finish

**Reply by davidpalmspringd** [reply]
Thanks. I’ll give it a shot.
